tests/unit/models/DefaultConfiguration.spec.js

```diff
@@ -39,5 +40,17 @@ describe('Test class: DefaultConfiguration', () => {
       expect(config.defaultFileExtension).toEqual('.test');
       expect(config.tags).toEqual(['test']);
     });
+
+    it('initializes ELK configuration properly', () => {
```

Comment:
add expect in other test instead of adding it.
Like:
`expect(config.elkParams).toEqual({...})`

Reply:
done